wlauto/workloads/googleplaybooks/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -64,12 +69,41 @@ public void runUiAutomation() throws Exception {
         switchPageStyles();
         aboutBook();
 
+        removeWatcher("pageSyncPop");
         pressBack();
         unsetScreenOrientation();
 
         writeResultsToFile(timingResults, parameters.getString("output_file"));
     }
 
+    // Creates a watcher for when a pop up warning appears when pages are out
+    // of sync across multiple devices.
+    private UiWatcher createPopUpWatcher() throws Exception {
+        UiWatcher pageSyncPopUpWatcher = new UiWatcher() {
+
+            @Override
+            public boolean checkForCondition() {
+                UiObject popUpDialogue = new UiObject(new UiSelector().resourceId("android:id/message"));
```

Comment:
As this just checks the resourceId isn't there a change this watcher will close any and all pop-ups?